### PR TITLE
enh(doc) add discovery rule for windows NRPE

### PR DESCRIPTION
**Pull request by cedricmeschin**
## Description

Adding NRPE discovery rule for disks on windows

## Target serie

- [x] 20.04.x
- [x] 20.10.x
- [x] 21.04.x
- [x] 21.10.x
- [x] 22.04.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 1]
```diff
@@ -15,6 +15,18 @@ monitoring agent and its embedded NRPE Server.
 * Windows Server OS from 2003 SP2 version
 * Windows Workstation from XP version
 
+### Discovery rules
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Services-->
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:22]:
```suggestion
**Services**
```

File: en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 2]
```diff
@@ -15,6 +15,18 @@ monitoring agent and its embedded NRPE Server.
 * Windows Server OS from 2003 SP2 version
 * Windows Workstation from XP version
 
+### Discovery rules
+
+<!--DOCUSAURUS_CODE_TABS-->
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:20]:
```suggestion
```

File: en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 3]
```diff
@@ -15,6 +15,18 @@ monitoring agent and its embedded NRPE Server.
...
+
+<!--Services-->
+
+| Rule name                 | Description |
+| :------------------------ | :---------- |
+| OS-Winfows-NRPE-Disk-Name |             |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:28]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 4]
```diff
@@ -19,6 +19,18 @@ monitoring agent and its embedded NRPE Server.
 * Windows Server OS from 2003 SP2 version
 * Windows Workstation from XP version
 
+### Discovery rules
+
+<!--DOCUSAURUS_CODE_TABS-->
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:24]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 5]
```diff
@@ -19,6 +19,18 @@ monitoring agent and its embedded NRPE Server.
...
+
+<!--Services-->
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:26]:
```suggestion
**Services**
```

File: fr/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md [hunk 6]
```diff
@@ -19,6 +19,18 @@ monitoring agent and its embedded NRPE Server.
...
+
+| Rule name                 | Description |
+| :------------------------ | :---------- |
+| OS-Winfows-NRPE-Disk-Name |             |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/operatingsystems-windows-nsclient-05-nrpe.md:32]:
```suggestion
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Comment by cedricmeschin** [comment]
Refactoring with the new format https://github.com/centreon/centreon-documentation/pull/1796

**Pull request closed without merging**